.changeset/large-beers-drive.md

```diff
@@ -0,0 +1,44 @@
+---
+'@keystone-ui/button': major
+'@keystone-ui/fields': major
+'@keystone-ui/icons': major
+'@keystone-ui/loading': major
+'@keystone-ui/modals': major
+'@keystone-ui/notice': major
+'@keystone-ui/options': major
+'@keystone-ui/pill': major
+'@keystone-ui/popover': major
+'@keystone-ui/segmented-control': major
+'@keystone-ui/toast': major
+'@keystone-ui/tooltip': major
+'@keystone-ui/website': major
+'@keystone-next/website': major
+'@keystone-next/example-auth': major
+'@keystone-next/app-basic': major
+'@keystone-next/example-ecommerce': major
+'@keystone-next/example-next-lite': major
+'@keystone-next/example-roles': major
+'@keystone-next/example-sandbox': major
+'@keystone-next/example-todo': major
+'@keystone-next/admin-ui': major
+'@keystone-next/admin-ui-utils': major
+'@keystone-next/auth': major
+'@keystone-next/cloudinary': major
+'@keystone-next/fields': major
+'@keystone-next/keystone': major
+'@keystone-next/session-store-redis': major
+'@keystone-next/types': major
+'@keystone-next/access-control-legacy': major
+'@keystone-next/adapter-prisma-legacy': major
+'@keystone-next/fields-legacy': major
+'@keystone-next/file-adapters-legacy': major
```

Comment:
```suggestion
```

These two packages are being removed in PRs that are about to land, so we don't want them going into any more changesets, since we won't be publishing them.